docs/config.md

```diff
@@ -304,14 +318,26 @@ model_names:
         batch_sizes: 1,2,3
 ```
 
+These parameters will result in testing the concurrency configurations of 2,
+10, 18, 26, 34, 42, 50, 58, and 64, for each of different batch sizes of 1, 2
+and 3. This will result in 27 individual test runs of the model.
+
 ## `<model-config-parameters>`
 
-This field represents the values that you want to change or sweep through
+This field represents the values that can be changed or swept through
 using Model Analyzer. All the values supported in the [Triton
 Config](https://github.com/triton-inference-server/server/blob/master/docs/model_configuration.md)
 can be specified or swept through here. `<model-config-parameters>`  should be
 specified on a per model basis and cannot be specified globally (like `<parameter>`).
 
+Table below presents the list of common parameters that can be used for manual sweeping:
+
+|       Option       |                                                 Description                                                  |
+| :----------------: | :----------------------------------------------------------------------------------------------------------: |
+| `dynamic_batching` |  https://github.com/triton-inference-server/server/blob/master/docs/model_configuration.md#dynamic-batcher   |
```

Comment:
Can you add the link to the Option Name and put a simple top level description in the description?